lib/client-assets.php

```diff
@@ -204,23 +204,23 @@ function gutenberg_register_vendor_scripts() {
 		array( 'react' )
 	);
 	gutenberg_register_vendor_script(
-		'tinymce-nightly',
-		'https://fiddle.azurewebsites.net/tinymce/nightly/tinymce' . $suffix . '.js'
+		'tinymce-latest',
+		'https://fiddle.azurewebsites.net/tinymce/4.6.5/tinymce' . $suffix . '.js'
 	);
 	gutenberg_register_vendor_script(
-		'tinymce-nightly-lists',
-		'https://fiddle.azurewebsites.net/tinymce/nightly/plugins/lists/plugin' . $suffix . '.js',
-		array( 'tinymce-nightly' )
+		'tinymce-latest-lists',
+		'https://fiddle.azurewebsites.net/tinymce/4.6.5/plugins/lists/plugin' . $suffix . '.js',
+		array( 'tinymce-latest' )
 	);
 	gutenberg_register_vendor_script(
-		'tinymce-nightly-paste',
-		'https://fiddle.azurewebsites.net/tinymce/nightly/plugins/paste/plugin' . $suffix . '.js',
-		array( 'tinymce-nightly' )
+		'tinymce-latest-paste',
+		'https://fiddle.azurewebsites.net/tinymce/4.6.5/plugins/paste/plugin' . $suffix . '.js',
+		array( 'tinymce-latest' )
 	);
 	gutenberg_register_vendor_script(
-		'tinymce-nightly-table',
-		'https://fiddle.azurewebsites.net/tinymce/nightly/plugins/table/plugin' . $suffix . '.js',
-		array( 'tinymce-nightly' )
+		'tinymce-latest-table',
+		'https://fiddle.azurewebsites.net/tinymce/4.6.5/plugins/table/plugin' . $suffix . '.js',
```

Comment:
Only thing I would have done differently is put the version number in one place so you can update it in one place.